PCF8591/pcf8591.c

```diff
@@ -41,11 +40,3 @@ return i2cread(ADC_CH_2);
 else if(ch == ADC_CH_3)
 return i2cread(ADC_CH_3);
```

Comment:
I can't figure out how to compile this one, in particular `i2cread` and friends seem to have incorrect captialisation if I try it with `I2C/i2c.h`?